es2015_exercise/readme.md

```diff
@@ -5,8 +5,8 @@ Convert the following es5 code blocks into es2015 code:
 ```javascript
 var person = {
     fullName: "Harry Potter",
-    sayHi: function(){
-        setTimeout(function(){
+    sayHi(){
+        setTimeout(() => {
             console.log("Your name is " + this.fullName)
         }.bind(this),1000)
```

Comment:
bind isn't needed because the arrow functions use the `this` from the surrounding area.  Also, the curly braces aren't needed for a 1 line arrow function.  So you can simplify to:

```
var person = {
    fullName: "Harry Potter",
    sayHi(){
        setTimeout(() => console.log("Your name is " + this.fullName),1000)
    }
}
```